### ISSUE TITLE
install sklearn error is shown

### ISSUE THREAD

**Issue by sherinmannambeth** [RasaHQ/rasa_nlu#521]
**rasa NLU version** (e.g. `0.7.3`):

**Used backend / pipeline** (`mitie`, `spacy_sklearn`, ...):

**Operating system** (windows, osx, ...):

**Issue**:

**Content of configuration file** (if used & relevant):
```json

```

**Comment by wrathagom** [comment]
How did you install Rasa?

**Comment by sherinmannambeth** [comment]
@wrathagom using the command pip install rasa_nlu

**Comment by wrathagom** [comment]
I remember seeing another issue recently with sklearn on windows. Do you get an error when you try to install the missing dependencies?

**Comment by sherinmannambeth** [comment]
Yes, I had "Failed building wheel for ...." errors. So I had to manually download the .whl files and install them, which seemed to work for everything except python-crfsuite.

**Comment by wrathagom** [comment]
Try to read through #362 see if it helps.

**Comment by sherinmannambeth** [comment]
@wrathagom Yes, I have installed Anaconda. #362 issue python version is different.

**Comment by wrathagom** [comment]
In a python shell can you import sklearn?

Also please try creating a clean conda environment and installing sklearn, does it succeed?

As far as I know very few of us have Windows, especially 8.1, so it will be hard for us to work on re-producing this.
